cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,31 @@
 
 describe('Book Store app', () => {
   before(() => {
-
+    cy.visit('/login');
   });
 
-  it('', () => {
+  it('Bookstore task', () => {
```

Comment:
Please, provide the name of the test case with more info. Also this task should contain few tests, you can not combine it into one (the requirements for auto tests are the same as for manual test cases, so, please, divide the tests to logical parts

Reply:
fixed